src/classes/issues-processor.ts

```diff
@@ -556,8 +557,7 @@ export class IssuesProcessor {
         (issue: Readonly<OctokitIssue>): Issue => new Issue(this.options, issue)
       );
     } catch (error) {
-      this._logger.error(`Get issues for repo error: ${error.message}`);
-      return Promise.resolve([]);
+      throw Error(`Get issues for repo error: ${error.message}`);
```

Comment:
```suggestion
      throw Error(`Getting issues was blocked by the error: ${error.message}`);
```
Could we also think about changing the rethrown error message to be more meaningful? I'm not sure that the current error message is helpful.

Reply:
I added the orignal backtrace to the re-thrown error

https://github.com/actions/stale/pull/935/files#diff-93e32a46e13f11c2066be36506b0f153d7da2b6b8d787da9177c10692ce50fc5R561